jvm-observ-lib/dashboards.libsonnet

```diff
@@ -0,0 +1,42 @@
+local g = import './g.libsonnet';
+{
+  new(this):
+    {
+      'jvm-dashboard.json':
+        g.dashboard.new(this.config.dashboardNamePrefix + 'JVM overview')
+        + g.dashboard.withVariables(this.signals.memory.getVariablesMultiChoice())
+        + g.dashboard.withTags(this.config.dashboardTags)
+        + g.dashboard.withUid(this.config.uid + '-jvm-dashboard')
+        + g.dashboard.withPanels(
+          g.util.panel.resolveCollapsedFlagOnRows(
+            g.util.grid.wrapPanels(
+              std.flattenArrays(
+                [
+                  this.grafana.rows.overview,
+                  this.process.grafana.rows.process,
```

Comment:
Somehow, this is causing the linting/compilation error where `panels` is not defined. I haven't been able to sort it out tho.

Reply:
thanks! I'll fix it

Reply:
there was a small bug in commonlib, which I had fixed here.